packages/create-apollo-client/test/__tests__/createApolloClient.test.js

```diff
@@ -0,0 +1,61 @@
+// @flow
+
+import { ApolloClient } from 'apollo-client';
+import { ApolloLink } from 'apollo-link';
+import { InMemoryCache, IntrospectionFragmentMatcher } from 'apollo-cache-inmemory';
+import { createApolloClient } from '../../src';
+
+jest.mock('apollo-client');
+jest.mock('apollo-link');
+jest.mock('apollo-cache-inmemory');
+
+
+describe('As a Developer, I can create 8base compability Apollo Client', () => {
+  beforeEach(() => {
+    jest.resetAllMocks();
+  });
+
+  it('should create Apollo Client with empty parameters', () => {
+    createApolloClient({})();
```

Comment:
I think you should check `createApolloClient`'s return value. 
```
const client = createApolloClient({})();
expect(client).toBeInstanceOf(ApolloClient);
```

Reply:
createApolloClient return function which is getting the schema and return Apollo client.
This is needs for the appollo-provider package.